README.md

```diff
@@ -220,7 +220,13 @@ $ find . -name "*.so" | xargs file
 For each x86-64 ELF file, check there is a corresponding aarch64 ELF file
 in the binaries. With some common packages (e.g. commons-crypto) we've seen that
 even though a JAR can be built supporting Arm manually, artifact repositories such as
-Maven don't have updated versions. Feel free to open an issue in this GitHub repo for advice.
+Maven don't have updated versions. To see if a certain artifact version may have Arm support,
+consult our [Common JARs with native code Table](CommonNativeJarsTable.md)
+which is kept up-to-date on best effort and through PRs.
```

Comment:
given it's a github repo it feels like this is redundant.

Reply:
Agree, fixed.